src/main/kotlin/kotlinx/kover/engines/intellij/IntellijCoverage.kt

```diff
@@ -53,6 +38,47 @@ internal fun Task.intellijReport(
     }
 }
 
+private fun Writer.writeModuleReportJson(
```

Comment:
Please add a comment, e.g. with a resulting JSON and its purpose for easier maintainability in the future